src/components/Chart/TimeBars.docs.md

```diff
@@ -756,3 +756,111 @@ year,value,color
   </ChartLegend>
 </div>
 ```
+
+### Ordinal Scale
+
+```react
+<div>
+  <ChartTitle>Die meisten Betriebe sind 10 bis 20 Hektaren gross</ChartTitle>
+  <ChartLead>Anteil der Betriebe nach Nutzfläche in Hektaren</ChartLead>
+  <CsvChart
+    config={{
+      "type": "TimeBar",
+      "x": "category",
+      "yTicks": [
+        0,
+        0.2,
+        0.4
+      ],
+      "domain": [
+        0,
+        0.4
+      ],
+      "color": "year",
+      "colorRange": [
+        "#004529"
+      ],
+      "labelFilter": "false",
+      "colorLegendValues": [
+        "2018"
+      ],
+      "xScale": "ordinal",
+      "unit": "der landwirtschaftlichen Betriebe",
+      "numberFormat": ".1%",
+      "xTicks": [
+        "< 1",
+        "1–3",
+        "3–5",
+        "5–10",
+        "10–20",
+        "20–30",
+        "30–50",
+        "> 50"
+      ]
```

Comment:
this overlaps on mobile:
<img width="333" alt="Screenshot 2021-06-15 at 12 44 23" src="https://user-images.githubusercontent.com/410211/122039875-6d478b00-cdd7-11eb-9bd6-109d88b19840.png">

Reply:
is this a problem code though or rather an example problem? part of me feels like if one wants to use this type of histogram, one should chose margins/labels/column number than will render ok on mobile too. (Which I clearly didn't do here – granted.)

Reply:
rotate 45° on mobile if needed.